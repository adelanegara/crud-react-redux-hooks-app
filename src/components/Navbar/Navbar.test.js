import React from "react";
import { screen, render } from "@testing-library/react";
import { NavbarUnwrapped } from "./index";

let wrapper;
beforeEach(() => {
  const { getByTestId } = render(<NavbarUnwrapped />);
  wrapper = getByTestId("navbar");
});

test("test navbar", () => {
  expect(wrapper).toBeInTheDocument();
});

test("Test title ABC Bank", () => {
  const wrapper = screen.getByText(/BANK ABC/);
  expect(wrapper).toBeInTheDocument();
});

test("Test title ABC Bank", () => {
  const wrapper = screen.getByText(/BANK ABC/);
  expect(wrapper).toMatchSnapshot();
});
