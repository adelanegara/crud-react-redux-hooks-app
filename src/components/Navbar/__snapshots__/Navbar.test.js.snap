// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test title ABC Bank 1`] = `
<div
  class="MuiTypography-root MuiTypography-h4 css-1yb0leh-MuiTypography-root"
>
  BANK ABC
   
</div>
`;
